src/Tests/InstrEngineTests/NaglerInstrumentationMethod/assertions.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+
+// Note, we use const char* because it is easier for cross-plat
+inline void AssertFailed(_In_ const char* message)
+{
+#ifdef DEBUG
+
+#ifdef PLATFORM_UNIX
+    // Just write to standard error
+    fprintf(stderr, "%s", message);
+#else
+    tstring wmessage;
+    SystemString::Convert(message, wmessage);
```

Comment:
>SystemString::Convert(message, wmessage); [](http://example.com/codeflow?start=4&length=41)

Missing `using "systemstring.h`

Reply:
That's not the issue. "systemstring.h" is included in stdafx. I forgot to qualify SystemString with the namespace.